object/search.go

```diff
@@ -285,6 +307,7 @@ func (f *SearchFilters) AddObjectIDFilter(m SearchMatchType, id oid.ID) {
 	f.addReservedFilter(m, fKeyObjectID, id)
 }
 
+// AddSplitIDFilter adds filter by split id.
```

Comment:
```suggestion
// AddSplitIDFilter adds filter by split ID.
```

OR a link to that type

related to every IDs in that PR, not just that line